packages/bpk-component-nudger/src/common-types.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+import PropTypes from 'prop-types';
+
+export type CommonProps<T> = {
+  id: string,
+  min: T,
+  max: T,
+  value: T,
+  onChange: T => void,
```

Comment:
This should be `onChange: T => mixed` imo since people might write a single line function that implicitly returns some value that we don't care about, but which will cause a flow error